services/fastfile_peeker/fastfile_peeker.rb

```diff
@@ -0,0 +1,55 @@
+require "fastfile_parser"
+require "digest"
+
+module FastlaneCI
+  # Utility class designed for parsing Fastfiles in a repo.
+  class FastfilePeeker
+    class << self
+      def cache
+        @cache ||= {}
+        return @cache
+      end
+
+      # Class method that finds the directory for the first Fastfile found given a root_path
+      # @param [String] root_path
+      # @return [String, nil] the path of the Fastfile or nil if no Fastfile found.
+      def fastfile_path(root_path: nil)
```

Comment:
We have very similar code inside `FastfileFinder` : https://github.com/fastlane/ci/blob/master/app/shared/fastfile_finder.rb#L16

Should we reference the `FastfileFinder` here or do something else?

Reply:
Didn't knew about that, gonna use it instead of this method